airflow/utils/helpers.py

```diff
@@ -376,3 +376,16 @@ def is_empty(x):
         return new_list
     else:
         return val
+
+
+def resolve_property_value(obj, prop):
+    """Return class property value.
+
+    :param obj: Reference to class.
+    :param prop: Reference to class property.
+    :returns: If ``prop`` property than return property value,
+        otherwise it returns class attribute value.
+    """
+    if isinstance(prop, property):
+        return prop.fget(obj)
+    return prop
```

Comment:
This can live directly in `serialized_objects` instead (and be a private function)

Reply:
Just to clarify - is it related to Operator?

Reply:
What do you mean? Not sure I understand the question.

Reply:
Sorry for confusing. Initially I think you mean add some implementation `__serialized_fields` in BatchOperator.
Now I understand what you mean place in `serialized_objects` module.